libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -103,6 +103,10 @@ interface IPhasedCommandTelemetry {
  * "build" script for each project.
  */
 export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
+  /**
+   * @internal
+   */
+  public _runsBeforeInstall: boolean | undefined;
```

Comment:
```suggestion
  public _runsBeforeInstall: boolean;
```

This also doesn't appear to ever be reassigned.

Reply:
It's not reassigned... in this repository. It's public with `@internal` because it is modified by the phased install plugin.